mozdownload/scraper.py

```diff
@@ -862,8 +862,8 @@ def platform_regex(self):
 
         PLATFORM_FRAGMENTS = {'linux': 'linux',
                               'linux64': 'linux64',
-                              'mac': 'macosx',
-                              'mac64': 'macosx64',
+                              'mac': r'macosx(64)?',
+                              'mac64': r'macosx(64)?',
```

Comment:
We don't need optional brackets here. For tinderbox builds there are only `macosx64` subfolders.